chapters/interface.tex

```diff
@@ -537,6 +537,11 @@ \section{Function-Compatibility or Function-Subtyping for Functions}\label{funct
   constraining interface of the function being redeclared.
 \end{itemize}
 
+Note that function-compatibility does currently not ensure that the function call has the correct variability, and that must then be checked after re-declaration -- due to the possibility of default arguments with higher variability.
+\begin{nonnormative}
+Thus any call of a function that is replaceable will need to be checked after redeclarations.
+\end{nonnormative}
+
```

Comment:
This looks like unnecessary repetition.  Can it simply be dropped?

Reply:
I dropped the non-normative texts that seemed redunddant and added a reference in the other part.

Reply:
I'm sure exactly what added reference you mean, but the unnecessary repetition seems fixed.  Resolving.